src/main/java/com/xpanxion/java/springboot/da1/demo/service/student1/GymService1.java

```diff
@@ -0,0 +1,29 @@
+package com.xpanxion.java.springboot.da1.demo.service.student1;
+
+import com.xpanxion.java.springboot.da1.demo.model.student1.Gym;
+import com.xpanxion.java.springboot.da1.demo.repository.instructor.student1.GymRepository1;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class GymService1 {
+
+    @Autowired
+    private GymRepository1 gymRepository1;
+
+    public void addGym(Gym gym) {
+
+        var id = gym.getId();
+        var name = gym.getName();
+        var street1 = gym.getStreet1();
+        var street2 = gym.getStreet2();
+        var city = gym.getCity();
+        var state = gym.getState();
+        var zip = gym.getZip();
+
+        Gym newGym = new Gym(id, name, street1, street2, city, state, zip);
```

Comment:
I don't think it's necessary to create a `newGym` object here.  We should be able to pass the gym into this:  `gymRepository1.save(gym);`  Can we try to remove the newGym and the vars above?

Reply:
Ok, Making changes now